README.md

```diff
@@ -457,6 +459,7 @@ Table of Contents
 ## Search
 
   * [algolia.com](https://www.algolia.com/) — Hosted search-as-you-type (instant). Free hacker plan up to 10,000 documents and 100,000 operations. Bigger free plans available for community/Open Source projects
+  * [Aleph](https://docs.alephdata.org/) — Aleph is a powerful and foss tool for people who follow the money. It helps investigators to securely access and search large amounts of data - no matter whether they are a government database or a leaked email archive.
```

Comment:
this doesnt appear to be a service? looks like they only provide self hosted options in that case we cant have it on the list I am afraid.

Others are great, thank you